charts/tempo-distributed/values.yaml

```
@@ -2192,6 +2194,30 @@ gateway:
           {{ . | nindent 4 }}
           {{- end }}
         }
+
+        {{- if .Values.gateway.enableGrpc }}
```

```suggestion
        {{- if .Values.traces.otlp.grpc.enabled }}
```
Using the same existing flag here as well to maintain consistency